x-pack/plugins/observability/server/index.ts

```diff
@@ -22,19 +22,26 @@ export { getInspectResponse } from '../common/utils/get_inspect_response';
 
 export * from './types';
 
+const configSchema = schema.object({
+  annotations: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
+    index: schema.string({ defaultValue: 'observability-annotations' }),
+  }),
+  unsafe: schema.object({
+    slo: schema.object({
+      enabled: schema.boolean({ defaultValue: false }),
+    }),
+  }),
+});
+
 export const config: PluginConfigDescriptor = {
   exposeToBrowser: {
     unsafe: true,
   },
```

Comment:
__question:__ just to double check, you need `unsafe` configuration block to be directly exposed to the browser part of you plugin?

Reply:
@azasypkin At the moment, we don't need anything exposed to the client side as we are focusing solely on the api. 

What does`exposeToBrowser` do? It exposes all the `unsafe.*` config to the browser? 😅

Reply:
Thanks for the explanation, I'm going to remove this